source/scss/_pf-basic-variables.scss

```diff
@@ -491,3 +491,8 @@ $state-danger-border:            $brand-danger;
 $alert-padding-x:             $spacer;
 $alert-link-font-weight:      normal;
 $pf-alert-icon-font-size:     $icon-size-sm;
+
+
+// Toast Notifications
+
+$pf-toast-notification-box-shadow-color: rgba($color-pf-black-1000, 0.2);
```

Comment:
I don't like this. lets create the shadow in a way that it can be reused as a whole. So we define 2 or 3 pf-shadows variables and those are the ones we reuse.